tests/resources/forecast_instance.json

```diff
@@ -0,0 +1,25 @@
+{
+  "spec": {
+    "office-id": "SPK",
+    "spec-id": "test-spec",
+    "designator": "designator",
+    "location-ids": [
+      "FcstInstTestLoc"
+    ]
+  },
+  "date-time": 1624284010000,
+  "issue-date-time": 1653221020000,
+  "first-date-time": 1692702150000,
+  "last-date-time": 1727017260000,
+  "max-age": 5,
+  "time-series-count": 3,
```

Comment:
The timeseries id's come back as a list for the spec. No need for a separate endpoint for the forecast timeseries since we already get the list of ids.

Reply:
Duh, I can read.